ziti/zitisock.py

```diff
@@ -0,0 +1,68 @@
+#  Copyright (c) 2022.  NetFoundry, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import socket
+from typing import Tuple
+from ziti import zitilib
+
+py_socket = socket.socket
+
+class ZitiSocket(py_socket):
```

Comment:
accept/bind/listen coming? do we need to support getsocketopts? does gethostbyname need to be overridden? maybe not needed yet?